EIPS/eip-712.md

```diff
@@ -1,23 +1,23 @@
 ---
 eip: 712
-title: Ethereum typed structured data hashing and signing
```

Comment:
Thank you for being open to it.

Since, IIUC the goal is to have a standardized way of hashing and signing, to avoid everyone re-inventing the wheels, I think it's more crypto-specific than crypto-generalized.

Option 1. *Ethereum typed structured data ~~hashing and~~ signing* (signing alone is likely to imply there will be some form of hashing)
Option 2. `Ethereum typed ~~structured~~ data hashing and signing` (typed usually imply structured)
Option 3. `Ethereum ~~typed~~ structured data hashing and signing` (structured usually imply typed)

Reply:
All EIPs are Ethereum Improvement Proposals. Ethereum is redundant in the title.